seed/breakfast-seeder.js

```diff
@@ -1,7 +1,7 @@
 const Breakfast = require('../models/breakfast');
 const mongoose = require('mongoose');
 
-mongoose.connect('mongodb://localhost:27017/chicken-fries',{ useNewUrlParser: true });
+mongoose.connect('mongodb://admin:root01@ds163402.mlab.com:63402/chicken-fries',{ useNewUrlParser:true});
```

Comment:
As with the earlier remark. Use environmental variables instead of hard-coding paths like this. Same goes for the other seed files.